### DISCUSSION TITLE
是否可以调整桌面歌词的位置和大小呢？

### DISCUSSION THREAD

**Discussion by Purga404** [zhongyang219/MusicPlayer2#791] [Ideas]
浏览网页的时候视线往往是向下滑，歌词在这个位置不太方便呢，希望可以放置顶端。如果悬浮歌词可以调节大小/字体/位置那就更好了。

**Comment by lrisora** [comment]
在windows，窗口只有非透明部分可以被点击
所以调整桌面歌词需要先关掉”歌词背景穿透“
字体字号可以在 设置->歌词设置 调整
